packages/jest-resolve/src/index.js

```diff
@@ -18,6 +18,8 @@ const path = require('path');
 const resolve = require('resolve');
 const browserResolve = require('browser-resolve');
 
+const {getModulePath} = require('jest-util');
```

Comment:
this should be in the same block as all the other requires, just move it on top of the block (because { precedes all characters in my sort plugin in Nuclide).